docs/experimental-machine-learning/host-risk-score.md

```diff
@@ -7,48 +7,63 @@ To deploy this framework in your environment, follow the steps outlined below.
 
 #### 1. Unzip the release bundle
 
-#### 2. Navigate to the Dev Tools console in Kibana
+#### 2. Modify artifacts to reflect Kibana space
+For security reasons, we require that you restrict the Host Risk Score application to a specific Kibana space. In order to do so, run the `generate_scripts.py` script with your Kibana space as the argument.
 
-#### 3. Uploading required scripts
+Eg:
 
-Upload the scripts in `ml_hostriskscore_levels_script.json`, `ml_hostriskscore_map_script.json` and `ml_hostriskscore_reduce_script.json` using the following API call:
+```
+python generate_scripts.py --space default
+```
+
+This will create a folder named `<your-space-name>`, consisting of all the required artifacts, modified for your Kibana space. **For the steps that follow, please use the scripts from this folder.**
+
+#### 3. Navigate to the Dev Tools console in Kibana
 
+You will now upload all the Host Risk Score artifacts to Kibana. Remember to suffix the names of all the scripts, ingest pipelines, transforms etc. with your Kibana space, while uploading them. 
+#### 4. Uploading required scripts
+
+Upload the contents of `ml_hostriskscore_levels_script.json`, `ml_hostriskscore_map_script.json` and `ml_hostriskscore_reduce_script.json` as individual scripts, using the Script API.
```

Comment:
These filenames have changed, I believe.

Reply:
In [dreml](https://github.com/elastic/dreml/tree/host_risk_score_7.16/releases/HostRiskScore) there are no files by these names, either before or after running `generate_scripts`. See my above comment about the need to copy those `.txt` files into the destination folder; we also need to update either those files or this document to be consistent.